controller-job.yaml

```diff
@@ -13,7 +13,13 @@ spec:
       containers:
       - name: sync-container
         image: gcr.io/k8s-jkns-gke-soak/secret-sync-controller:latest
-        command: ["sh", "-c", "go build -o controller-cmd cmd/controller/main.go && ./controller-cmd --config-path /tmp/config/syncConfig --period 1000"]
+        command: 
+        - sh
+        args:
+        - -c
+        - |
+          go build -o secret-sync-controller cmd/secret-sync-controller/main.go
+          ./secret-sync-controller --config-path /tmp/config/syncConfig --period 1000
```

Comment:
I may be wrong but I think `go build ./cmd/secret-sync-controller` will get you the same result

Reply:
This is correct in general cases.
However it would yield error `go build: build output "secret-sync-controller" already exists and is a directory` because there's a `secret-sync-controller` folder in the current directory (`/workspace`) .